pkg/handler/events/log_events.go

```diff
@@ -0,0 +1,48 @@
+package events
+
+import (
+	"github.com/sirupsen/logrus"
+	"github.com/water-hole/ansible-operator/pkg/runner/eventapi"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+type LogLevel int
+
+const (
+	Everything LogLevel = iota
+	Tasks
+	Nothing
+)
+
+type EventHandler interface {
+	Handle(*unstructured.Unstructured, eventapi.JobEvent) error
+}
+
+type loggingEventHandler struct {
+	LogLevel LogLevel
+}
+
+func (l loggingEventHandler) Handle(u *unstructured.Unstructured, e eventapi.JobEvent) error {
```

Comment:
It looks like this could never return an `error`. Should it just not have a return value?

Reply:
I think your right. We will be fanning these `EventHandler` out and not waiting for there return. I think forcing the Handle function to deal with its errors make sense to me. 

Changing this now.